src/components/AuthLogin/BreakLine.tsx

```diff
@@ -0,0 +1,15 @@
+import React from "react";
+
+const BreakLine = ({ label }: { label: string }) => {
+  return (
+    <div className="flex whitespace-nowrap w-full justify-center items-center">
+      <hr className="w-full border-tertiary" />
+      <span className="m-2 w-min rounded-lg text-white text-sm uppercase">
```

Comment:
```suggestion
      <span className="mx-2 w-min rounded-lg text-white text-xs uppercase">
```

This should make the height 16px